### PR TITLE
243 all molecule names

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: R/sensitivity-analysis-run-options.R [hunk 1]
```diff
@@ -18,6 +18,15 @@ SensitivityAnalysisRunOptions <- R6::R6Class(
       super$initialize(ref)
       self$numberOfCoresToUse <- numberOfCoresToUse
       self$showProgress <- showProgress
+    },
+    #' @description
+    #' Print the object to the console
+    #' @param ... Rest arguments.
+    print = function(...) {
```
Comment (msevestre) [inline R/sensitivity-analysis-run-options.R]: @PavelBal Also added for sa run options

File: R/simulation-run-options.R [hunk 2]
```diff
@@ -21,6 +21,16 @@ SimulationRunOptions <- R6::R6Class(
       self$numberOfCoresToUse <- numberOfCoresToUse
       self$checkForNegativeValues <- checkForNegativeValues
       self$showProgress <- showProgress
+    },
+    #' @description
+    #' Print the object to the console
+    #' @param ... Rest arguments.
+    print = function(...) {
```
Comment (msevestre) [inline R/simulation-run-options.R]: @copy from your code

Comment (copy) [reply R/simulation-run-options.R]: I don't think I wrote this.

Comment (msevestre) [reply R/simulation-run-options.R]: Sorry this was meant to be @PavelBal  copy. Cheers

**Pull request merged**